turbJet/turbJet/turbJet.usr

```diff
@@ -88,12 +76,29 @@ C-----------------------------------------------------------------------
       include 'NEKUSE'
       include 'NEKNEK'
 
-      integer e, bID
+      integer e, bID, ixy
+      real vn, U0, delta, S0
 
       e = gllel(ieg)
       bID = boundaryIDList(iside,e)
- 
-      if (bID.eq.3 .or. bID.eq.4) then ! coflow + lateral
+      
+c     Dong boundary condition
+      pa = 0.0
+      U0 = 1.0                  ! characteristic velocity
+      delta = 0.1               ! small positive constant
+      if (cbu.eq.'o  ') then
+         if (iside.eq.1.or.iside.eq.3) then
```

Comment:
Can you please explain what this does?

Reply:
you have to calculate normal velocity component on the outflow only, so you have to identify outflow, and the position in un[xyz] array

Reply:
I don't like "heavy" code like this in the usr file. May be we should compute the face normal in FACEIS() and FACEIV() and store it in NEKUSE variable. Another option could be to introduce a subroutine to compute the local face normal like fnormal(un,ix,iy,iz,iel,iface). May be it exisits already?

Probably useful

Reply:
It does already exist, it's getSnormal(un,ix,iy,iz,iel,iface).